tools/src/main/scala/scala/scalanative/interflow/MergeProcessor.scala

```diff
@@ -18,7 +18,8 @@ final class MergeProcessor(
         local -> offset
     }.toMap
   val blocks = mutable.Map.empty[Local, MergeBlock]
-  var todo = mutable.Set.empty[Local]
+  val todoOrdering: Ordering[Local] = Ordering.by(offsets)
+  var todo = mutable.SortedSet.empty[Local](todoOrdering)
```

Comment:
It probably should be immutable field with mutable collection. 
```suggestion
  val todo = mutable.SortedSet.empty[Local](todoOrdering)
```